docs/README.md

```diff
@@ -0,0 +1,3 @@
+# Building the Documentation #
+
+To build the documentation to `repobee` simply follow the steps that can be found [Here!](https://repobee.readthedocs.io/en/stable/contributing.html#contributing-to-docs)
```

Comment:
As this is developer documentation, it should link to latest. And put in a hard line wrap at ~80 characters, plz :)

```suggestion
To build the documentation to `repobee` simply follow the steps that can be found
[Here!](https://repobee.readthedocs.io/en/latest/contributing.html#contributing-to-docs)
```